pdns/dnsdistdist/docs/guides/serverselection.rst

```diff
@@ -45,7 +45,7 @@ For example, if we have two servers, with respective weights of 1 and 4, we expe
 ``whashed`` is a similar weighted policy, but assigns questions with identical hash to identical servers, allowing for better cache concentration ('sticky queries').
 The current hash algorithm is based on the qname of the query.
 
-.. function:: setWHashedPertubation(value)
+.. function:: setWHashedPerturbation(value)
 
   Set the hash perturbation value to be used in the whashed policy instead of a random one, allowing to have consistent whashed results on different instances.
```

Comment:
```suggestion
.. function:: setWHashedPerturbation(value)

  .. versionchanged:: 2.0.0
    Before 2.0.0 this function was misspelled as `setWHashedPertubation`

  Set the hash perturbation value to be used in the whashed policy instead of a random one, allowing to have consistent whashed results on different instances.
```